nvflare/fuel/f3/streaming/byte_streamer.py

```diff
@@ -230,7 +220,7 @@ def _stop_task(self, task: TxTask, error: StreamError = None, notify=True):
                 task.stream_future.set_exception(error)
 
             if notify:
-                message = Message(None, None)
+                message = Message(task.headers)
```

Comment:
Should make a copy of these headers, instead of reusing task.headers, to be safe.

Reply:
Changed.